seq-mainnet-1/RUN_VALIDATOR.md

```diff
@@ -0,0 +1,285 @@
+
+- `host`: host for the gRPC server to listen on
+- `port`: port for the gRPC server to listen on
+- `eth_ws_url`: Ethereum node WebSocket endpoint
+- `eth_rpc_url`: Ethereum node RPC endpoint
+- `eth_contract_address`: address in hex format of the contract to monitor for logs. This MUST be set to `0xBa0e6bF94580D49B5Aaaa54279198D424B23eCC3`.
+- `sequencer_grpc_url`: Sequencer node gRPC endpoint
+
+### Linux
+
+On Linux, you can use `systemd` to run the Sequencer in the background. Knowledge of how to use `systemd` is assumed here.
+
+Here's an example service file with some placeholder (`<...>`) values that must be filled-in:
+
+<details>
+  <summary>Click me...</summary>
+
+```sh
+[Unit]
+Description=Sidecar
+After=network.target
+
+[Service]
+Type=simple
+User=<USER>
+ExecStart=<HOME>/go/bin/fuelsequencerd start-sidecar \
+    --host "0.0.0.0" \
+    --sequencer_grpc_url "127.0.0.1:9090" \
+    --eth_ws_url "<ETHEREUM_NODE_WS>" \
+    --eth_rpc_url "<ETHEREUM_NODE_RPC>" \
+    --eth_contract_address "0xBa0e6bF94580D49B5Aaaa54279198D424B23eCC3"
+Restart=on-failure
+RestartSec=3
+LimitNOFILE=4096
+
+[Install]
+WantedBy=multi-user.target
+```
+</details>
+
+### Mac
+
+On Mac, you can use `launchd` to run the Sequencer in the background. Knowledge of how to use `launchd` is assumed here.
+
+Here's an example plist file with some placeholder (`[...]`) values that must be filled-in:
+
+<details>
+  <summary>Click me...</summary>
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>fuel.sidecar</string>
+
+    <key>ProgramArguments</key>
+    <array>
+        <string>/Users/[User]/go/bin/fuelsequencerd</string>
+        <string>start-sidecar</string>
+        <string>--host</string>
+        <string>0.0.0.0</string>
+        <string>--sequencer_grpc_url</string>
+        <string>127.0.0.1:9090</string>
+        <string>--eth_ws_url</string>
+        <string>[ETHEREUM_NODE_WS]</string>
+        <string>--eth_rpc_url</string>
+        <string>[ETHEREUM_NODE_RPC]</string>
+        <string>--eth_contract_address</string>
+        <string>0xBa0e6bF94580D49B5Aaaa54279198D424B23eCC3</string>
+    </array>
+
+    <key>UserName</key>
+    <string>[User]</string>
+
+    <key>KeepAlive</key>
+    <dict>
+        <key>SuccessfulExit</key>
+        <false/>
+    </dict>
+
+    <key>HardResourceLimits</key>
+    <dict>
+        <key>NumberOfFiles</key>
+        <integer>4096</integer>
+    </dict>
+
+    <key>StandardOutPath</key>
+    <string>/Users/[User]/Library/Logs/fuel-sidecar.out</string>
+    <key>StandardErrorPath</key>
+    <string>/Users/[User]/Library/Logs/fuel-sidecar.err</string>
+</dict>
+</plist>
+```
+</details>
+
+## Restart the Sequencer
+
+Assuming your Sequencer node is already running, it should be restarted at this point given that we have reconfigured some values.
```

Comment:
Yeah this should be avoided ideally to avoid confusion. That's why I am suggesting one guide with separate instructions for full nodes and validators, extracting the exact common parts at the top.

Reply:
Otherwise if you want to have them in separate guides I would suggest copying over some instructions from the full node guide.